test/basic.test.ts

```diff
@@ -0,0 +1,15 @@
+import { describe, it, expect } from 'vitest'
+import { fileURLToPath } from 'node:url'
+import { setup, $fetch } from '@nuxt/test-utils'
+
+describe('ssr', async () => {
+  await setup({
+    rootDir: fileURLToPath(new URL('./fixtures/basic', import.meta.url)),
```

Comment:
/cc @antfu Can we use global setup with `nuxi test` now?

Reply:
It's a module starter, I guess it means to be tested with different fixtures? The global setup works better for application tests.

Reply:
Yes, definitely different fixtures in the case of modules